vendor/github.com/containerd/imgcrypt/images/encryption/client.go

```diff
@@ -44,8 +44,15 @@ func WithDecryptedUnpack(data *imgcrypt.Payload) diff.ApplyOpt {
 			return fmt.Errorf("failed to marshal payload: %w", err)
 		}
 
+		// typeurl.MarshalAny currently returns &types.Any, but it will
+		// return typeurl.Any soon.
+		pbany := &types.Any{
+			TypeUrl: any.GetTypeUrl(),
+			Value:   any.GetValue(),
+		}
+
 		for _, id := range imgcrypt.PayloadToolIDs {
-			c.ProcessorPayloads[id] = any
+			c.ProcessorPayloads[id] = pbany
 		}
```

Comment:
This is the local change I made. If that looks okay, I'm going to open a PR against imgcrypt.

Reply:
Thanks! Opened https://github.com/containerd/imgcrypt/pull/72.